pages/create.js

```diff
@@ -11,7 +12,8 @@ const StyledBackLink = styled(StyledLink)`
 export default function CreatePlacePage() {
   const router = useRouter();
 
-  function addPlace(place) {
+  export default function addPlace(place) {
+    const {mutate} = use
```

Comment:
you probably meant to destructure mutate() from useSWR

I recommend doing it at the top level of your component (for instance right below the line where your call useRouter)